src/app/filecoin-plus/allocators/components/FilterableAllocatorsTable.tsx

```diff
@@ -0,0 +1,57 @@
+'use client'
+
+import {
+  useReactTable,
+  getCoreRowModel,
+  getSortedRowModel,
+  getFilteredRowModel,
+} from '@tanstack/react-table'
+
+import { SearchInput } from '@/components/SearchInput'
+
+import { allocatorsTableColumnDef } from '../data/allocatorsTableColumnDef'
+import { type Allocator } from '../schemas/AllocatorSchema'
+
+import { AllocatorsTable } from './AllocatorsTable'
+import { NoFilterResultsMessage } from './NoFilterResultsMessage'
+
+type FilterableAllocatorsTableProps = {
+  data: Array<Allocator>
+}
+
+export function FilterableAllocatorsTable({
```

Comment:
What if we call this component `FilterContainer` like we do for the main shared `FilterContainer` layout?

Reply:
That's true. What about `AllocatorTableWithFilters`?

Reply:
Let's do `AllocatorTableWithFilters`!

Reply:
> It might end up looking very similar to FilterContainer

The key difference here is that `AllocatorTableWithFilters` holds the state for the table and filters, while `FilterContainer` is a real stateless container that can contain anything.

So, we may need to keep `AllocatorsTableWithFilters` and create another proper container component for the layout of our table filters. We can do that in the next PR when we add sorting.